### PR TITLE
Docker-compose file to power up a postgresql database

### PR DESCRIPTION
**Pull request by yazanessam**

### CODE REVIEW

**Comment by Mahmoudgalalz** [comment]
LGTM

**Review by Mahmoudgalalz — APPROVED** [review]

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+version: '3'
+
+services:
+  database:
+    image: 'postgres:latest'
+    environment:
```
Comment (0xHunterr) [inline docker-compose.yaml:7]: Predictable , weak credentials

Comment (yazanessam) [reply docker-compose.yaml:7]: @Mahmoudgalalz so ?

Comment (Mahmoudgalalz) [reply docker-compose.yaml:7]: @0xHunterr this is Just for Develompent I have asked for them like that

**Pull request merged**